pom.xml

```diff
@@ -97,6 +97,16 @@
       <artifactId>quarkus-junit5</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
```

Comment:
Should be test scoped?

Reply:
Yep. Done.

Reply:
Are you still planning to push a commit here?

Reply:
Incoming 🚀